bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "cache.open-cluster-management.io/v1",
+          "kind": "OCMSearch",
+          "metadata": {
+            "name": "search-v2-operator",
+            "namespace": "open-cluster-management"
+          },
+          "spec": null
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: search-v2-operator.v2.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: OCMSearch is the Schema for the ocmsearches API
+      displayName: OCMSearch
+      kind: OCMSearch
+      name: ocmsearches.cache.open-cluster-management.io
+      version: v1
+  description: Operator to install ACM Search
```

Comment:
Should we change "ACM Search" to "OCM Search" here for consistency?

Reply:
Done